Libplanet/Action/ActionEvaluator.cs

```diff
@@ -483,9 +483,10 @@ ActionContext CreateActionContext(IAccountStateDelta prevStates, int randomSeed)
                     .ForContext("Tag", "Metric")
                     .ForContext("Subtag", "TxEvaluationDuration")
                     .Debug(
-                        "{ActionCount} actions in transaction {TxId} by {Signer} " +
+                        "{ActionCount} actions {ActionType} in transaction {TxId} by {Signer} " +
                         "with timestamp {TxTimestamp} evaluated in {DurationMs:F0}ms.",
                         tx.Actions.Count,
+                        tx.Actions.FirstOrDefault(),
```

Comment:
```suggestion
                        "{ActionCount} actions {ActionTypes} in transaction {TxId} by {Signer} " +
                        "with timestamp {TxTimestamp} evaluated in {DurationMs:F0}ms.",
                        tx.Actions.Count,
                        tx.Actions.Select(action => action.ToString().Split(".").LastOrDefault()),
```

Not the best solution, but something like this might work. 🙄

Reply:
Sorry for frequent commenting but now that I think of it, `ToString()` might not be enough. Try `GetType().Name` or something similar.

Reply:
![image](https://user-images.githubusercontent.com/42176649/162950587-6c9e74a1-9937-4239-8b41-3f7c3603ab5b.png)
`GetType().Name` returns `PolymorphicAction1` so we might need to stick with `ToString()` 😢. It would be nice if there was a way to access the `innerAction` name value but I haven't found a way to do that.

Reply:
My bad, now I see that we need to get the generic type.